### DISCUSSION TITLE
3D Print settings

### DISCUSSION THREAD

**Discussion by momonunu** [stevenbennett/DIYson#20] [Q&A]
Are there any recommended setting for printing the parts such as layer heights, infill and wall thickness?

**Comment by stevenbennett** [comment]
@momonunu Yeah this is something I've been meaning to do. I will try to write up a little text-based guide tonight that covers the basics.

**Reply by momonunu** [reply]
Cool! I am planning to do an assembly guide video next week when my parts will arrive.

**Reply by stevenbennett** [reply]
Wow awesome, looking forward to seeing that so please share when you're done :)

Very basic print guide just added in a820998abeef2d2be7770c44c0e754854316be1a

**Comment by stevenbennett** [comment]
Print guide added in a820998abeef2d2be7770c44c0e754854316be1a
